modules/smarthubBidAdapter.js

```diff
@@ -16,12 +16,12 @@ const ALIASES = [
   {code: 'vimayx', skipPbsAliasing: true},
 ];
 const BASE_URLS = {
-  smarthub: 'https://prebid.smart-hub.io/pbjs',
-  markapp: 'https://markapp-prebid.smart-hub.io/pbjs',
-  jdpmedia: 'https://jdpmedia-prebid.smart-hub.io/pbjs',
-  tredio: 'https://tredio-prebid.smart-hub.io/pbjs',
-  felixads: 'https://felixads-prebid.smart-hub.io/pbjs',
-  vimayx: 'https://vimayx-prebid.smart-hub.io/pbjs',
+  smarthub: 'https://prebid.attekmi.com/pbjs',
+  markapp: 'https://markapp-prebid.attekmi.com/pbjs',
+  jdpmedia: 'https://jdpmedia-prebid.attekmi.com/pbjs',
+  tredio: 'https://tredio-prebid.attekmi.com/pbjs',
+  felixads: 'https://felixads-prebid.attekmi.com/pbjs',
+  vimayx: 'https://vimayx-prebid.attekmi.com/pbjs',
 };
```

Comment:
Why don't you add Attekmi as another alias and add docs for it?

Reply:
We are currently rebranding the entire company, and changed Smarthub to Attekmi. We also leave simultaneous support for old *-prebid.smart-hub.io endpoints and new *-prebid.attekmi.con endpoints for current connections.

Reply:
Why don't you add support for that alias?